File: app/src/main/java/org/amoustakos/boilerplate/examples/io/local/models/ExampleModel.java
package org.amoustakos.boilerplate.examples.io.local.models;

import io.realm.RealmModel;
import io.realm.annotations.RealmClass;


@RealmClass
public class ExampleModel implements RealmModel{


	public static final String COL_ID = "id",
								COL_NAME = "name";


	public String id;
	public String name;



}

File: app/src/main/java/org/amoustakos/boilerplate/examples/io/local/models/ExampleModel.kt
package org.amoustakos.boilerplate.examples.io.local.models

import io.realm.RealmModel
import io.realm.annotations.RealmClass

@RealmClass
data class ExampleModel (
                                var id: String? = null,
                                var name: String? = null
                        ) : RealmModel {

    companion object Column {
        val ID = "id"
        val NAME = "name"
    }


}